templates/web/base/maps/openlayers.html

```diff
@@ -33,4 +33,6 @@
     [% IF map.copyright %]
     <div class="olControlAttribution" style="position: absolute;">[% map.copyright %]</div>
     [% END %]
-
+    <div id="loading-indicator" class="hidden" aria-hidden="true">
+      <img src="/i/loading.svg" alt="Loading.." />
```

Comment:
Three dots for an ellipsis ;) And if we're editing this line anyway, might as well have the normal four space indent too.